### ISSUE TITLE
Yaml shorthand !Fn::Sub

### ISSUE THREAD

**Issue by martysweet** [martysweet/cfn-lint#10]
```yaml
      UserData:
        - Fn::Base64
            !Fn::Sub "PERSCODE=${ProvisionKey}"
```

Does not throw any validation errors

!Fn is invalid, should be !Sub
